app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -203,6 +203,12 @@
                 {{input type=field.type value=(mut (get data.speaker field.fieldIdentifier))
                         id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))}}
               {{/if}}
+            {{else if (eq field.fieldIdentifier 'sponsorshipRequired')}}
+              {{ui-checkbox label=(t 'Sponsorship is required') checked = data.speaker.isNotRequired 
+              onChange = (action (mut data.speaker.isNotRequired))}} 
+              {{#if (not data.speaker.isNotRequired)}}
+                {{input type='hidden' id='speaker_sponsorshipRequired_required' value=(mut (get data.speaker field.fieldIdentifier))}} 
+              {{/if}}
```

Comment:
Change the type of `sponsorshipRequired` to select in `Form Mixins`.

Reply:
After making new changes :
[https://youtu.be/pN3ZL35Bb6g](https://youtu.be/pN3ZL35Bb6g)

Reply:
> Change the type of `sponsorshipRequired` to select in `Form Mixins`.

@kushthedude i changed that to select and implemented that but that is case for dropdown and in case of checkbox when only two value are needed i think this might be good .
please tell me if i am missing something ?

Reply:
Drop-down implementation As  #3180 , changed sponsorShiprequired to 'select' in `form mixins`  then added utils/dictionary/sponsorshiprRequired 
```
export const sponsorships = [
  {
    name: 'Yes'
  },
  {
    name: 'No'
  }
];
```
![Screenshot from 2019-12-28 17-50-33](https://user-images.githubusercontent.com/56407566/71543585-a8af8900-299a-11ea-8626-58cefafdc828.png)

Reply:
I just gave you reference, You don't have to implement it same. Also, you have to make `Yes/No` two radio checkboxes.

Reply:
> @kushthedude i changed that to select and implemented that but that is case for dropdown and in case of checkbox when only two value are needed i think this might be good .

Nope, Select means you have already given options and value needs to be of Yes or No.

Reply:
> I just gave you reference, You don't have to implement it same. Also, you have to make `Yes/No` two radio checkboxes.

one checkbox is not good instead of two radio-button ?

Reply:
https://youtu.be/pN3ZL35Bb6g , its working correctly .